docs/commands/mftf.md

```diff
@@ -468,6 +468,7 @@ Runs all MFTF static-checks on the test codebase that MFTF is currently attached
 #### Existing static checks
 
 * Test Dependency: Checks that test dependencies do not violate Magento module's composer dependencies.
+* Unused Arguments: Checks that action groups do not have unused arguments.
```

Comment:
How about document the desired behavior regarding arguments in action group merging and extending cases?

Reply:
As discussed, can check if documentation is necessary if it's not intuitive to users at a later point.